CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/).
 ### Fixed
 - `Autocomplete` Fixed an issue where re-entering the autocomplete field would trigger an async search with empty query
 - `Autocomplete` Fixed an issue where debounce was using old query values
+- `Textarea` Fixed an issue where required property was not being passed properly
```

Comment:
This should go under `[Unreleased]` ;-)

Reply:
Ah oke I thought this version was unreleased